grizli/pipeline/auto_script.py

```diff
@@ -4572,7 +4572,8 @@ def make_filter_combinations(root, weight_fnu=2, filter_combinations=FILTER_COMB
             head[band] = im_i[0].header.copy()
         else:
             for k in im_i[0].header:
-                head[band][k] = im_i[0].header[k]
+                #head[band][k] = im_i[0].header[k]
```

Comment:
You can also just skip the offending keys:
`if k == '' or k == 'HISTORY': continue`

Reply:
sounds good, I can change that